Lib/venv/__init__.py

```diff
@@ -178,6 +179,29 @@ def create_configuration(self, context):
             f.write('version = %d.%d.%d\n' % sys.version_info[:3])
             if self.prompt is not None:
                 f.write(f'prompt = {self.prompt!r}\n')
+            f.write('executable = %s\n' % os.path.realpath(sys.executable))
+            args = []
+            nt = os.name == 'nt'
+            if nt and self.symlinks:
+                args.append('--symlinks')
+            if not nt and not self.symlinks:
+                args.append('--copies')
+            if not self.with_pip:
+                args.append('--without-pip')
+            if self.system_site_packages:
+                args.append('--system-site-packages')
+            if self.clear:
+                args.append('--clear')
+            if self.upgrade:
+                args.append('--upgrade')
+            if self.upgrade_deps:
+                args.append('--upgrade-deps')
+            if self.orig_prompt is not None:
+                args.append(f'--prompt={self.orig_prompt}')
```

Comment:
```python
                    prompt = f'"{self.orig_prompt}"' if ' ' in self.orig_prompt else self.orig_prompt
                    args.append(f'--prompt={prompt}')
```
to handle the case where `self.orig_prompt` has spaces?

Reply:
I think it's not enough to quote it on spaces, I quoted it unconditionally in case of chars like '*'.

Reply:
Ah, I just realized I did not commit that change.. pushing it now.